sql/parser/Makefile

```diff
@@ -0,0 +1,3 @@
+sql.go: sql.y
+	go tool yacc -o sql.go sql.y
+	@echo "$$(echo '// Code generated by go yacc.' | cat - sql.go)" > sql.go
```

Comment:
Let me know if there is a better/cleaner way to add a line to the top of an existing file.